recipe/scripts/activate-dpcpp.bat

```diff
@@ -1,2 +1,5 @@
 set "CC=icx"
 set "CXX=icx"
+
+set "INCLUDE=%CONDA_PREFIX%\include;%INCLUDE%"
+set "LIB=%CONDA_PREFIX%\Library\lib;%LIB%"
```

Comment:
This makes these directories in BUILD_PREFIX have higher priority than PREFIX, which is wrong. Not sure what to do here.

Reply:
@isuruf could you please elaborate on the issue? Will it help if we add these directories to the end of the list instead of adding at the beginning?